proxy-node-gateway/src/test/java/uk/gov/ida/notification/resources/HubResponseResourceTest.java

```diff
@@ -87,13 +95,14 @@ public void testsHappyPath() {
             "eidas_relay_state_in_session"
         );
 
-        when(session.getAttribute(SESSION_KEY_SESSION_DATA)).thenReturn(sessionData);
+        when(sessionCacheManager.getSession(ArgumentMatchers.anyString())).thenReturn(sessionData);
```

Comment:
Could we use String "session-id" instead of `ArgumentMatchers.anyString()` to make the test more meaningful?